utils/reqApi.js

```diff
@@ -100,6 +107,28 @@ class ABRequestAPI {
       this._user = u;
    }
 
+   get userReal() {
+      return this._userReal;
+   }
+
+   set userReal(u) {
+      this._userReal = u;
+   }
+
+   isSwitcherood() {
+      return this._userReal != null;
+   }
+
+   /**
+    * @method switcherooToUser()
+    * allow the current user to impersonate the provided user.
+    * @param {json:SiteUser} u
+    */
+   switcherooToUser(u) {
+      this.userReal = this.user;
```

Comment:
Should are these params `this._userReal` and `this._user`?

Reply:
I chose to reuse the other getters/setters so that incase I change how the user & userReal data is stored, this function doesn't need to be updated.